src/ghazal_azadi_ex/ex01/tidy_code.py

```diff
@@ -0,0 +1,45 @@
+from random import randint as rand_num
+
+__author__ = 'ghazalazadi'
+__email__ = 'ghaz@nmbu.no'
+
+
+def guessed_num():
+    """Returns the number being guessed by the user
+    """
+    inp_num = 0
+    while inp_num < 1:
+        inp_num = int(input('Your guess: '))
+    return inp_num
+
+
+def true_num():
+    """Returns 2 * random number (between 1,6)  being created by the randint function
+    """
+    return rand_num(1, 6) + rand_num(1, 6)
+
+
+def check_fun(f, g):
```

Comment:
`f` and `g` have no meanings here to be the names of function arguments. Can you think of something better?